user_interface/lambda_function.py

```diff
@@ -155,35 +164,40 @@ def create_new_reminder(user_reminder: UserReminder, name: str) -> None:
     ])
     bot.send_message(
         user_reminder.user_id, 
-        text=f'You started to create reminder called {name}. Please, choose '
+        text=f'You started to create reminder called {name}. Choose '
              f'number of times per day you want to repeat a task. '
-             f'Use command:\n "/set_amount Number"')
+             f'Please, use command:\n "/set_amount Amount"')
 
 
-def set_reminder_number(user_reminder: UserReminder, number: str) -> None:
+def set_reminder_amount(user_reminder: UserReminder, number: str) -> None:
     """Sets reminder number in table temp_reminder
     
     :param number: amount of reminders per day
     :param user_reminder: UserReminder object for current user
     :return: None
     """
-    chat_id = user_reminder.user_id
-    if number.strip().isdigit():
+    user_id = user_reminder.user_id
+    if user_reminder.temp_reminder.name == DEFAULT_NAME:
+        bot.send_message(
+            user_id, 
+            text=f'Please, start from command /create ReminderName')
+    
+    elif number.strip().isdigit():
         cur_temp_reminder = user_reminder.temp_reminder
         cur_temp_reminder.amount = int(number)
         cur_temp_reminder.current_state = int(number)
         user_reminder.update(actions=[
             UserReminder.temp_reminder.set(cur_temp_reminder)
         ])
         bot.send_message(
-            chat_id, text=f'Please, choose frequency of remineders per day. '
-                        f'Reminders will be send from 9 till 21 your time with '
-                        f'a frequency you will choose. For example, 12 means '
-                        f'every hour. '
-                        f'Use command:\n "/set_frequency Frequency"')
+            user_id, text=f'Please, choose frequency of remineders per day. '
+                        f'Reminders will be sent between 9 am and 9 pm your time with '
+                        f'a frequency you will choose. For example, 1 means '
+                        f'every hour, 2 means every 2 hours and so on. '
+                        f'Please, use command:\n "/set_frequency Frequency"')
     else:
-        bot.send_message(chat_id, text=f'Input ({number}) is not a number. '
-                        f'Use command:\n "/set_amount Number"')
+        bot.send_message(user_id, text=f'Input "{number}" is not a number. '
+                        f'Please, use command:\n "/set_amount Amount"')
         
 
 def set_reminder_frequency(user_reminder: UserReminder, frequency: str) -> None:
```

Comment:
Whenever the main argument to a function is an instance, try to think about if the function makes sense to be a method on the class of the instance instead.

For example, this function could be a part of the `UserReminder` class which you would then call by `user_reminder.set_frequency(frequency)`